server/src/main/resources/routes

```diff
@@ -3,7 +3,10 @@
 # ~~~~
 
 # API routes
--> /api/v1  de.frosner.broccoli.ApiV1Router
+-> /api/v1  de.frosner.broccoli.routes.ApiV1Router
+
+# Downloads
+-> /downloads/ de.frosner.broccoli.routes.DownloadsRouter
```

Comment:
> While we are certainly using the endpoint through a browser this is just temporary.

If it's just temporary what's going to replace it? What will replace this endpoint to provide the ability to, well, "download" a snapshot of the log to view (grep, less, etc.) it on the local system?

I'm aware that we'll stream logs at some point but is that a replacement for the ability to get the log on the local system?  I can't `grep` in the webui, can I?

> But what you get there is a resource and the protocol should be versioned.  Why do I care? If someone uses Broccoli from a script or from the command line and we change the API […]

The resource offers logs as they are on nomad node.  How could this endpoint change?  Or, in other words, if we wanted to change this endpoint wouldn't that rather be a change in the semantics and thus be a wholly new endpoint?

> I didn't fully get the separate URL prefix issue. Each endpoint can return 404 individually. If you request an instance that does not exist, you get 404. Same for logs.

Try `curl localhost:9000/this-does-not-exist` versus `curl localhost:9000/downloads/this-does-not-exist`:  The latter returns an empty 404, whereas the former returns a HTML page.

Since we already have a top-level catch-all route (for assets) that's only possible with a separate prefix.  

For the same reason there's a dedicated API router now.